sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathFunctionsSuite.scala

```diff
@@ -95,6 +94,19 @@ class MathFunctionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     checkEvaluation(c(Literal(1.0), Literal.create(null, DoubleType)), null, create_row(null))
   }
 
+  test("conv") {
```

Comment:
are you saying this is what hive does? i feel it is better to be more strict here, e.g. "11abc" just returns null if the base is 10.

However, I don't have a strong preference here.